packages/app-desktop/gui/styles/dialog-modal-layer.scss

```
@@ -20,11 +20,13 @@
 		color: var(--joplin-color);
 		padding: 16px;
 		box-shadow: 6px 6px 20px rgba(0,0,0,0.5);
-		margin: 20px;
 		min-height: fit-content;
 		display: flex;
 		flex-direction: column;
 		border-radius: 10px;
+
+		margin-left: auto;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)